flexmeasures/ui/templates/crud/asset_audit_log.html

```diff
@@ -13,6 +13,7 @@ <h3>History of actions for asset <a href="/assets/{{ asset.id }}">{{ asset.name
             <table id="asset_audit_log" class="table table-striped table-responsive paginate nav-on-click" title="View data">
                 <thead>
                     <tr>
+                        <th style="display:none;">Event Timestamp</th><!-- Hidden UTC Timestamp column for sorting, Keep at position zero -->
                         <th>Event Datetime</th>
```

Comment:
I noticed that we can order by this visible Event Datetime column, but that would then lead to the ordering we don't want. Do you know of a way to not sort by a column?
In any case, we don't have to do this improvement in this PR.

Reply:
Yes that's a good observation, we can reroute sorting to the hidden Time Stamp when 'Event Datetime' column is clicked. this can be done inside the `<script> ... </script>`. I'll do this in a separate PR.